detection-rules/brand_impersonation_tiktok.yml

```
@@ -0,0 +1,76 @@
+name: "Brand Impersonation: TikTok"
+description: "Detects messages impersonating TikTok through similar display names or logo detection, combined with security-themed content and authentication failures. Excludes legitimate TikTok communications and trusted senders."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    // display name contains tiktok
+    (
+      strings.ilike(strings.replace_confusables(sender.display_name), '*tiktok*')
+      // levenshtein distance similar to tiktok
+      or strings.ilevenshtein(strings.replace_confusables(sender.display_name),
+                              'tiktok'
+      ) <= 1
+      or any(ml.logo_detect(beta.message_screenshot()).brands,
+             .name == "TikTok" and .confidence == "high"
+      )
+    )
+  )
+  and (
+    any(beta.ml_topic(body.current_thread.text).topics,
+        .name in (
+          "Security and Authentication",
+          "Secure Message",
+          "Reminders and Notifications"
+        )
+        and .confidence in ("medium", "high")
+    )
+    or any(beta.ml_topic(beta.ocr(beta.message_screenshot()).text).topics,
+           .name in (
+             "Security and Authentication",
+             "Secure Message",
+             "Reminders and Notifications"
+           )
+           and .confidence in ("medium", "high")
+           and beta.ocr(beta.message_screenshot()).text != ""
+    )
+    or any(ml.nlu_classifier(body.current_thread.text).intents,
+           .name == "cred_theft" and .confidence == "high"
+    )
+    or any(ml.nlu_classifier(beta.ocr(beta.message_screenshot()).text).intents,
+           .name == "cred_theft" and .confidence == "high"
+    )
+  )
+  
+  // and the sender is not in org_domains or from tiktok domains and passes auth
+  and not (
+    sender.email.domain.root_domain in $org_domains
+    or (
+      sender.email.domain.root_domain in ("tiktok.com", "tiktokglobalshop.com")
```

https://platform.sublime.security/messages/eb6e2439fc47187adfc4ae8b842338622f8850542cba7b20cd1620be50ca74e8?preview_id=0195ec77-ac0e-7556-a2c0-ebf230deac2f
```suggestion
      sender.email.domain.root_domain in ("tiktok.com", "tiktokglobalshop.com", "bytedance.com")
```